ts/screens/messages/paginated/MessageDetailScreen.tsx

```diff
@@ -147,7 +145,7 @@ const MessageDetailScreen = ({
 };
 
 const mapStateToProps = (state: GlobalState, ownProps: OwnProps) => {
-  const message: UIMessage = ownProps.navigation.getParam("message");
+  const message: UIMessage = ownProps.route.params.message;
```

Comment:
@fabriziofff what if we use this PR to fix this? We might pass just `message.id` and `message.serviceId` as param instead of the entire object.

Reply:
We can assume that message exists in the store at this point (otherwise you cannot be here). Try to load it with the `getById` selector. If it's undefined we simply show an error screen.